renderer/components/Directory.jsx

```diff
@@ -1,76 +1,94 @@
-import React from 'react';
+import React, { Component } from 'react';
+import cx from 'classnames';
 import File from './File.jsx';
 import CreateMenu from './CreateMenu.jsx';
 import CreateForm from './CreateForm.jsx';
 import RenameForm from './RenameForm.jsx';
 
-export default class Directory extends React.Component {
-  constructor() {
-    super();
-  }
+export default class Directory extends Component {
+  handleListItemClick = proxyEvent => (
```

Comment:
You're wrong. Arrow functions are easy to read. They help us to avoid using `bind` in `constructor`.
When we are writing arrow functions we do not need `constructor` and `bind`, because arrow functions already bound into an instance.

Just read - https://medium.com/@housecor/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56